hypersync-net-types/src/log.rs

```diff
@@ -623,4 +647,27 @@ mod tests {
 
         Ok(())
     }
+
+    #[test]
+    fn test_log_filter_topic_configurations() {
+        fn check_log_filter_json(json: serde_json::Value) {
+            let lf: LogFilter = serde_json::from_value(json).unwrap();
+            println!("{lf:?}");
+            let mut message = capnp::message::Builder::new_default();
+            let mut lf_builder =
+                message.init_root::<hypersync_net_types_capnp::log_filter::Builder>();
+            lf.populate_builder(&mut lf_builder).unwrap();
+            let lf_reader = lf_builder.into_reader();
+            let read_lf: LogFilter = LogFilter::from_reader(lf_reader).unwrap();
+            assert_eq!(lf, read_lf);
+        }
+
+        const TOPIC: &str = "0x1234567890123456789012345678901234567890123456789012345678901234";
+
+        check_log_filter_json(json!({"topics": []}));
+        check_log_filter_json(json!({"topics": [[], [], [], []]}));
+        check_log_filter_json(json!({"topics": [[], [], [], []]}));
+        check_log_filter_json(json!({"topics": [[TOPIC], [], [], []]}));
+        check_log_filter_json(json!({"topics": [[], [], [TOPIC]]}));
```

Comment:
Did some manual tests, then I was worried about what else could have issues. So I added fuzz tests for the whole query. All is working well.